File: Source Code/DemoOnlineFloralDelivery/DemoOnlineFloralDelivery/Service/CommentService.cs
using DemoOnlineFloralDelivery.Models;

namespace DemoOnlineFloralDelivery.Service;

public interface CommentService
{
    //Hiển thị danh sách toàn bộ Comment
    public dynamic findAllComment();

    public dynamic findByBouquetId(int bouquetId);

    //Thêm(add) Comment
    public bool addComment(Comment _comment);
    //Cập nhật(update) Comment
    public bool updateComment(Comment comment);
    //Xóa(Delete) Comment
    public bool deleteComment(int CommentId);
}

File: Source Code/AdminOnlineFloralDelivery/src/app/service/comment.service.ts
import { Injectable } from "@angular/core";
import { BaseURLService } from "./baseurl.service";
import { HttpClient } from "@angular/common/http";
import { lastValueFrom } from "rxjs";


@Injectable()
export class CommentService{
 constructor(
  private baseURLService: BaseURLService,
  private httpClient: HttpClient
 ){}

 async findAll(){
  return await lastValueFrom(this.httpClient.get(this.baseURLService.getBaseUrl()+'comment/findAllComment'));
}

async findByBouquetId(boquetId : string){
  return await lastValueFrom(this.httpClient.get(this.baseURLService.getBaseUrl()+'comment/findByBouquetId/'+boquetId));
}

async delete(commentId: string){
  return await lastValueFrom(this.httpClient.delete(this.baseURLService.getBaseUrl()+'comment/deleteComment/'+commentId));
}

}
